new-client/src/plugins/Kir/KirSearchView.js

```diff
@@ -0,0 +1,399 @@
+              </Typography>
+              <Grid container spacing={0} alignItems="center">
+                <Grid item xs={6}>
+                  <Slider
+                    value={this.state.ageValues}
+                    onChange={this.handleAgeChange}
+                    valueLabelDisplay="off"
+                    aria-labelledby="range-slider"
+                    step={1}
+                    min={0}
+                    max={this.state.maxAge}
+                  />
+                </Grid>
+                <Grid item xs={3} className={classes.ageInputContainer}>
+                  <TextField
+                    fullWidth
+                    variant="outlined"
+                    size="small"
+                    value={this.state.ageValues[0]}
+                    className={classes.input}
+                    onChange={this.inputMinAgeChanged}
+                    inputRef={this.inputMinAge}
+                    inputProps={{
+                      step: 1,
+                      min: 0,
+                      max: this.state.maxAge,
+                      type: "number",
+                    }}
+                  />
+                </Grid>
+                <Grid item xs={3} className={classes.ageInputContainer}>
+                  <TextField
+                    fullWidth
+                    variant="outlined"
+                    size="small"
+                    value={this.state.ageValues[1]}
+                    className={classes.input}
+                    onChange={this.inputMaxAgeChanged}
+                    inputRef={this.inputMaxAge}
+                    inputProps={{
+                      step: 1,
+                      min: 0,
+                      max: this.state.maxAge,
+                      type: "number",
+                    }}
+                  />
+                </Grid>
+              </Grid>
+            </div>
+
+            <div
+              className={classes.containerTopPadded}
+              style={{ textAlign: "right" }}
+            >
+              <Button
+                variant="outlined"
+                color="primary"
+                component="span"
+                size="small"
+                onClick={this.handleClearSearch}
+                className={classes.clearButton}
+              >
+                Rensa
+              </Button>
+              <Button
+                variant="contained"
+                color="primary"
+                component="span"
+                size="small"
+                onClick={this.handleSearch}
+                disabled={this.state.loading}
+              >
+                Sök
+                {this.state.loading && (
+                  <CircularProgress
+                    size={24}
+                    className={classes.buttonProgress}
+                  />
+                )}
+              </Button>
+            </div>
+          </AccordionDetails>
+        </Accordion>
+      </>
+    );
+  }
+}
+
+const styles = (theme) => ({
```

Comment:
Not much of a comment, rather a thought: this amount of custom styling will make it increasingly difficult for you to migrate to MUI5 styling (if not difficult than at least require more rewriting). I'd try not to style as much and go with the default looks wherever possible. The defaults are usually very sane.

Reply:
I might keep this as is, and get back to it later after launch if that's ok.